code/game/objects/items/devices/radio/radio_analog.dm

```diff
@@ -11,16 +11,16 @@
 		return FALSE
 	if (wires.IsIndexCut(WIRE_RECEIVE))
 		return FALSE
-	if(!(0 in level) && !(get_z(src) in level))
+	if(!(0 in z_levels) && !(get_z(src) in z_levels))
 		return FALSE
 	return TRUE
 
 /proc/broadcast_analog_radio_message(datum/radio_frequency/connection, mob/speaker,
 	obj/item/radio/radio, message, intercom_only = FALSE,
-	hard_to_hear, list/levels, verbage = "says", decl/language/speaking = null, list/secured
+	hard_to_hear, list/z_levels, verbage = "says", decl/language/speaking = null, list/secured
 	)
 
-	var/list/radios = list()
+	var/list/radios = list(radio)
```

Comment:
I think it was an intentional change that the broadcasting radio wasn't included, but I'm fine with it being readded.